src/services/token.service.js

```diff
@@ -0,0 +1,29 @@
+const { Token } = require('../models/Token.model');
+
+const save = async (userId, newToken) => {
+  const token = await Token.findOne({ where: { UserId: userId } });
+
+  if (!token) {
+    await Token.create({ UserId: userId, refreshToken: newToken });
+
+    return;
+  }
+
+  token.refreshToken = newToken;
+
+  await token.save();
+};
+
+const getByToken = async (refreshToken) => {
+  return Token.findOne({ where: { refreshToken } });
+};
+
+const remove = async (userId) => {
+  return Token.destroy({ where: { UserId: userId } });
```

Comment:
Ensure that the 'UserId' field matches the actual field name in your database schema. Sequelize typically uses camelCase for field names unless explicitly defined otherwise.